src/api/app/views/webui2/webui/package/show.html.haml

```diff
@@ -12,9 +12,9 @@
       .col-md-8
         %h3#package-title
           = @package.title.presence || @package.name
-          - if @package.url.present?
-            %small= link_to(@package.url, @package.url)
-        #description-text
+        - if @package.url.present?
+          %small.text-muted= link_to(@package.url, @package.url)
+        .mt-3#description-text
```

Comment:
Is the margin top also needed if the package doesn't have a URL? I'm not sure, but I would think it should be on the link itself (or its container `%small`)

Reply:
I would also put it in the link